Temporary fix for https://github.com/voidlinux/void-packages/issues/2425

@dalias affirmed that returning UTF-8 even when requesting C locale is just fine.

--- src/locale/langinfo.c.orig	2015-09-01 08:41:59.830797631 +0200
+++ src/locale/langinfo.c	2015-09-01 08:42:15.324966247 +0200
@@ -34,7 +34,7 @@ char *__nl_langinfo_l(nl_item item, loca
 	const char *str;
 
 	if (item == CODESET)
-		return MB_CUR_MAX==1 ? "UTF-8-CODE-UNITS" : "UTF-8";
+		return "UTF-8";
 	
 	switch (cat) {
 	case LC_NUMERIC:
